bin/mdsctl

```diff
@@ -76,7 +76,7 @@ declare -A configs=(
   ["values"]="${MDS_VALUES:-}"
   ["values-mds"]="${MDS_VALUES_MDS:-}"
   ["presets"]="${MDS_PRESETS:-}"
-  ["presets-mds"]="${MDS_PRESETS_MDS:-pgPass=Password123#,apis.mds-agency.migration=true}"
+  ["presets-mds"]="${MDS_PRESETS_MDS:-postgresql.password=Password123#,apis.mds-agency.migration=true}"
```

Comment:
`Password123` incredible

Reply:
Also, why do we not care that the password is set here? Does this only apply when PG is running in the cluster and hence we're not concerned?

Reply:
@drtyh2o correct - this only gets used when deploying postgresql internal to kubernetes.  Even in that case, it can be overridden if desired.

Reply:
Password123# is intentionally bad, and we should make it clear in the README (if not already) that it's for testing and development only